packages/frontend/src/components/nft/NFTTransferModal.js

```diff
@@ -0,0 +1,397 @@
+            const res = await NonFungibleTokens.transfer({
+                accountId: owner_id,
+                contractId: contract_id,
+                tokenId: token_id,
+                receiverId
+            });
+
+            onTransferSuccess(res, receiverId);
+        } catch (err) {
+            dispatch(showCustomAlert({
+                success: false,
+                messageCodeHeader: 'error',
+                messageCode: 'walletErrorCodes.sendNonFungibleToken.error',
+                errorMessage: err.message,
+            }));
+        } finally {
+            setSending(false);
+        }
+    }
+
+    return (
+        <Modal
+            id='nft-transfer-modal'
+            isOpen={nft}
+            onClose={onClose}
+            closeButton={false}
+            modalSize='md'
+        >
+            {viewType === 'transfer' &&
+            <StyledContainer className='small-centered'>
+                <img className='transfer-img' src={nft.metadata.mediaUrl} alt='NFT'/>
+
+                <h3><Translate id='NFTTransfer.transferNft'/></h3>
+                <p className='transfer-txt'><Translate id='NFTTransfer.enterReceipt'/></p>
+
+                <form>
+                    <div className='receiver-input'>
+                        <ReceiverInputWithLabel
+                            receiverId={receiverId}
+                            handleChangeReceiverId={receiverId => setReceiverId(receiverId)}
+                            checkAccountAvailable={accountId => dispatch(checkAccountAvailable(accountId))}
+                            localAlert={localAlert}
+                            autoFocus={!isMobile()}
+                            clearLocalAlert={() => dispatch(clearLocalAlert())}
+                            setAccountIdIsValid={setAccountIdIsValid}
+                        />
+                    </div>
+
+                    <ModalFooter>
+                        <div className='buttons'>
+                            <FormButton
+                                className='link'
+                                type='button'
+                                onClick={onClose}
+                                color='gray'
+                            >
+                            <Translate id='NFTTransfer.cancel'/>
+                            </FormButton>
+                            <FormButton
+                                className='next-btn'
+                                type='submit'
+                                disabled={!accountIdIsValid}
+                                onClick={() => setViewType('confirm')}
+                            >
+                            <Translate id='NFTTransfer.next'/>
+                            </FormButton>
+                        </div>
+                    </ModalFooter>
+                </form>
+            </StyledContainer>
+            }
+
+            {viewType === 'confirm' &&
+                <StyledContainer className='small-centered'>
```

Comment:
```suggestion
                <StyledContainer>
```
`small-centered` isn't doing anything here